### ISSUE TITLE
Support ESLint v9

### ISSUE THREAD

**Issue by koba04** [wikimedia/eslint-docgen#138]
Do you have a plan to support ESLint v9?

**Comment by edg2s** [comment]
I haven't had time to start working with ESL9 config, but patches are welcome.

**Comment by koba04** [comment]
Thanks. To support ESLint v9, upgrading ESLint plugins and configs (`wikimedia`, `node` etc) might be necessary.

**Comment by edg2s** [comment]
In theory one can use FlatCompat to load legacy configs, but I couldn't get it to work easily.

**Comment by koba04** [comment]
> In theory one can use FlatCompat to load legacy configs, but I couldn't get it to work easily.

Yeah, I tired it, but I got the error and to fix this, updating the `wikimedia` config seems necessary

> Error: Failed to load plugin 'yml' declared in ' » eslint-config-wikimedia#overrides[1] » ./yaml': This method cannot be used with flat config. Add your entries directly into the config array.
